testapp1.py

```diff
@@ -29,8 +29,8 @@ def decode_jwt_token(jwt_token, public_key):
         return None
 
 
-@app.route("/health")
-async def health_check():
+@app.route("/ping")
+async def ping():
```

Comment:
we need to standardize across all backend.

/health made sense to me as we could want to offer more information than ok (loading, stopping, exception, etc)

Reply:
done